src/main/java/org/shanerx/tradeshop/enumys/ShopStatus.java

```diff
@@ -31,7 +33,7 @@
 	CLOSED("&c");
 
 	private String colour;
-	private static final char COLOUR_CHAR = '&';
+	private transient static final char COLOUR_CHAR = '&';
```

Comment:
I know i was just being paranoid, because it also shouldnt serialize parent classes that dont extend serializable but it was trying to serialize utils. changed

Reply:
I think it might be indeed supposed to serialize all fields of an object unless instructed otherwise. Inherited objects fully belong to the extending class, which is probably why Gson was attempting to serialize the Plugin instance.

Reply:
I saw it somewhere but i cant find it now super classes shouldn't be serialized unless they also extend serializable, but that doesn't matter anymore since its not extending it